vendor/tracing/zero/destinations.ml

```diff
@@ -32,7 +32,69 @@ let direct_file_destination ?(buffer_size = 4096 * 16) ~filename () =
   (module Dest : Writer_intf.Destination)
 ;;
 
-let file_destination ~filename () = direct_file_destination ~filename ()
+let compressed_file_destination ?(buffer_size = 4096 * 16) ~filename () =
+  let buf = Iobuf.create ~len:buffer_size in
+  let com_size = buffer_size * 2 in
```

Comment:
Why does the compressed `Iobuf` size need to be 2&times; the size of of the regular buffer? Can we leave a comment here?

Reply:
Left a comment!